src/ui/component/userVerify/view.jsx

```diff
@@ -25,15 +26,16 @@ class UserVerify extends React.PureComponent<Props> {
   }
 
   render() {
-    const { errorMessage, isPending, verifyPhone } = this.props;
+    const { errorMessage, isPending, verifyPhone, fetchUser } = this.props;
     return (
       <React.Fragment>
         <section className="section__header">
           <h1 className="section__title--large">{__('Extra Verification Needed')}</h1>
           <p>
             {__(
-              "We weren't able to auto-approve you for rewards. Please complete one of the steps below to unlock them."
+              "We weren't able to auto-approve you for rewards. Please complete one of the steps below to unlock them. Then you can"
             )}{' '}
+            <Button onClick={() => fetchUser()} button="link" label={__('check again.')} /> {'Or'}{' '}
```

Comment:
I think this can just say "Refresh". Then we don't need the change to the other text.